scripts/sct_crop_image.py

```diff
@@ -87,8 +87,8 @@ def get_parser():
         '-xmax',
         type=int,
         default=-1,
-        help="Higher bound for cropping along X. Setting '-1' will crop to the maximum dimension, '-2' will crop to "
-             "the maximum dimension minus 1 slice, etc.",
+        help="Higher bound for cropping along X. Setting '-1' will crop to the maximum dimension (i.e. no change), "
+             "'-2' will crop to the maximum dimension minus 1 slice, etc.",
```

Comment:
zmin should be: "Lower bound for cropping along Z.",
zmax should be: "Higher bound for cropping along Z. Follows the same rules as xmax."